### ISSUE TITLE
Sketcher: "Translate parameters" in array function incorrectly says "Copies" rather than "Columns"

### ISSUE THREAD

**Issue by naich58** [FreeCAD/FreeCAD#17399]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

![Screenshot 2024-10-22 115144](https://github.com/user-attachments/assets/4a42b5f3-db0e-4f4d-92c9-3257fcb0cfcd)

When "Move / Array transform" is selected on a highlighted item, the "Translate parameters" box has the "Colums" box marked as "Copies".

### Full version info

```shell
OS: Windows 11 build 22631
Word size of FreeCAD: 64-bit
Version: 1.0.0RC2.38806 (Git)
Build type: Release
Branch: (HEAD detached at 1.0rc2)
Hash: 3d63fc6c2f665a8d5e6468845a419bcac80756c7
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: English/United Kingdom (en_GB)
Stylesheet/Theme/QtStyle: unset/FreeCAD Classic/Qt default
Installed mods: 
  * Assembly4 0.12.2
  * Manipulator 1.4.9
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
@PaddleStroke is it really columns or is copies correct?

**Comment by PaddleStroke** [comment]
Well both are corrects I guess. I'm ok with both. It would make sense to use columns since we use rows under. I used copies because if you just want to make a linear pattern then copies makes sense.

Also if you leave 0 in the copies, it just moves existing geos, which is conceptually sound. If we changed to column, then column = 0 wouldn't make much sense.

Either way is ok for me. Perhaps others can weight in.

**Comment by maxwxyz** [comment]
yes, columns makes no sense if you want to just transform the selection. Then 0 copies would be correct.
@FreeCAD/design-working-group any input?

**Comment by pierreporte** [comment]
It’s not clear at first sight. Maybe with a radio button to either just translate or make an array (same for rotate and scale). Overall I don’t really like the UX for this. I feel that they are two very different operations and should be separate features.

**Comment by luzpaz** [comment]
What's the verdict here ?

**Comment by luzpaz** [comment]
soft bump

**Comment by naich58** [comment]
I guess we stay with "Copies" then. Seems a pity to keep another thing that makes people go "huh?".

**Comment by Sloloem** [comment]
@naich58 @pierreporte I actually would like to second the idea of these being separate tools again.  It feels like the combination of the 2 tools is actually trying to operate in different "modes" to do different things that seem like unrelated concepts from a user's perspective.  It makes the combination tool really unintuitive to use for any of 3 possible use-cases I can see, and that's assuming a user even understands enough about what the tool does to realize it covers those use-cases.

"Copies" really only makes sense when the tool is used to make a linear pattern.  From a user perspective that's easy to conceptualize as "X instances in a line", but comes with the caveat that `copies=instances-1` and "rows" is just UI clutter for the linear pattern mode.  That said, it also would make just as much sense as "1 row of X columns" and wouldn't be nearly as confusing for making grids.

Neither parameter is really required to use it as a move tool, but it forces the user to understand that "move" as an action is actually a "0-copy linear array".  It's like a text editor without a cut command but somehow a way to paste something -1 times.  I understand _why_ 0 copies moves geometry, but it seems like a strange way to conceptualize "moving".

And for a grid "copies" makes users have to translate a 2x2 grid to a "1-copy linear array then make 2 rows of it".  It's kinda like making a grid with Part Design's multi-transform where you make perpendicular linear arrays, but you still have to remember `copies=columns-1`.

Ever since 1.0 I mostly changed my workflow to avoid the sketcher and do arrays with the Draft Array tools instead.  The usability is better even if you have to run the math out yourself if you want the rectangular array to spread out at angles other than 90°, the parameters all make sense.  I immediately understood what the tool was going to do before I used it.  Move is its own tool to not confuse the issue.

**Comment by FEA-eng** [comment]
Still relevant

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.2.0dev.20260429 (Git shallow)
Build date: 2026/04/29 07:07:16
Build type: Release
Branch: grafted,grafted
Hash: a785d40ac4c37f559b0b82fbaf9bf7dc68a468e4
Python 3.11.15, Qt 6.8.3, Coin 4.0.8, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL) [ OS: Polish/Poland (pl_PL) ]
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Navigation Style/Orbit Style/Rotation Mode: Blender/Rounded Arcball/Window center
Logical DPI/Physical DPI/Pixel Ratio: 96/113.268/1.25
```
